randovania/game_connection/connector_builder_choice.py

```diff
@@ -14,13 +14,14 @@ class ConnectorBuilderChoice(Enum):
     NINTENDONT = "nintendont"
     DREAD = "dread"
     CS = "cave-story"
+    AM2R = "am2r"
 
     @property
     def pretty_text(self) -> str:
         return _pretty_backend_name[self]
 
     def is_usable(self) -> bool:
-        if self is ConnectorBuilderChoice.DEBUG:
+        if self is ConnectorBuilderChoice.DEBUG or self is ConnectorBuilderChoice.AM2R:
```

Comment:
Add a ToDo to remember to remove it later

Reply:
hmmm, since these are only available in dev builds right now anyway, and i plan to get this into the next stable release, i might as well remove it now.